tests/test_ruby.sh

```diff
@@ -43,4 +52,25 @@ function test_rbenv {
   done
 }
 
+function test_rvm {
+
+  rvm-prompt() {
+    printf '%s\n' "ruby-3.2.2"
+  }
+
+  rvm() {
+    printf '%s\n' "$__rvm_output"
+  }
```

Comment:
`liquidprompt` never calls `rvm`. So if your `rvm-prompt` mock doesn't either, then there is no point in mocking `rvm`.

Also, the `rvm-prompt` mock shouldn't be hard coded. We have the array of inputs, use that.

```suggestion
  rvm-prompt() {
    printf '%s\n' "$__rvm_output"
  }
```